### PR TITLE
adds support for Health Icons

### LINKED ISSUE #73: add support for Health Icons
[https://healthicons.org](Health Icons) are CC0 licensed icons intended for use in health care related projects. It would be helpful to add support to this package for these icons.

I would be happy to look at implementing this if you feel like this would be a suitable addition to the package.

### PR DESCRIPTION
**Pull request by tomjemmett**
Resolves #73 by adding support for the Health Icons set. This adds the function `download_health_icons()` and the icon set `health_icons`.

Health Icons provide a single download file which is the most recent set of icons. It does not appear as though they have an archive for previous versions, so there is no version argument to `download_health_icons`

It appears as though this is most similar to Google material for the way to search for icons, so I have copied that pattern. The following work:
- `health_icons("blood_bag")`
- `health_icons("blood_bag", theme = "negative")`
- health_icons("blood_bag", category = "blood", theme = "negative")

however, providing a category but no theme fails
- `health_icons("blood_bag", category = "blood")`

Does this feel like it's in keeping with the rest of the package? Happy to make any changes necessary.

### CODE REVIEW

**Comment by Moohan** [comment]
I did a quick check of this, using it in a Shiny app and it seemed to work as expected!

File: R/health_icons.R [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+#' @export
+download_health_icons <- function(){
+  url <- "https://healthicons.org/icons.zip"
+
+  meta <- jsonlite::read_json(
+    "https://raw.githubusercontent.com/resolvetosavelives/healthicons/main/package.json"
+  )
+
+  install_icon_zip(
+    "health_icons", url, function(dl_dir) file.path(dl_dir, "icons", "svg"),
```
Comment (mitchelloharawild) [inline R/health_icons.R:12]: I've made a small fix here, the zip contains a `__MACOSX` folder which needs careful handling.

File: R/health_icons.R [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+}
+
+#' Font Awesome icons
+#'
+#' @param name Name of the icon
+#' @param category The icon's category, either "blood", "body", "conditions",
+#'   "devices", "diagnostics", "emotions", "family", "medications", "objects",
+#'   "people", "places", "shapes", "specialties", "symbols", "typography", or
+#'   "vehicles"
+#' @param theme The style variant for the icon, requires v4.0.0 or greater.
```
Comment (mitchelloharawild) [inline R/health_icons.R:30]: I've updated the themes to match what is available for this icon set.

**Review by mitchelloharawild — COMMENTED** [review]
Looks great, thanks for your contribution!
If you'd like, could you please add yourself to the DESCRIPTOIN as a `ctb` author?

Thanks also for figuring out how to extend the package despite no extension documentation being written yet. Well done! :tada:

File: R/health_icons.R [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+      name = "Health Icons",
+      version = meta$version,
+      licence = c("CC-0", "MIT")
+    )
+  )
+
+  invisible(health_icons)
+}
+
+#' Font Awesome icons
```
Comment (Moohan) [inline R/health_icons.R]: I think this is a typo.
```suggestion
#' Health icons
```

Comment (tomjemmett) [reply R/health_icons.R]: oops, fixed that!

**Comment by tomjemmett** [comment]
added myself to DESCRIPTION! 🙂

**Review by mitchelloharawild — APPROVED** [review]
Looks good, thanks for your PR!

**Pull request merged**